Cannot find module '/app/dist/src/index.js'

Logs from Heroku

```plain
2020-04-29T18:05:54.388868+00:00 heroku[web.1]: State changed from starting to crashed
2020-04-29T18:05:54.267866+00:00 app[web.1]: 
2020-04-29T18:05:54.267884+00:00 app[web.1]: > @ start /app
2020-04-29T18:05:54.267884+00:00 app[web.1]: > node dist/src/index.js
2020-04-29T18:05:54.267885+00:00 app[web.1]: 
2020-04-29T18:05:54.318242+00:00 app[web.1]: internal/modules/cjs/loader.js:960
2020-04-29T18:05:54.318243+00:00 app[web.1]: throw err;
2020-04-29T18:05:54.318244+00:00 app[web.1]: ^
2020-04-29T18:05:54.318244+00:00 app[web.1]: 
2020-04-29T18:05:54.318244+00:00 app[web.1]: Error: Cannot find module '/app/dist/src/index.js'
2020-04-29T18:05:54.318245+00:00 app[web.1]: at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
2020-04-29T18:05:54.318245+00:00 app[web.1]: at Function.Module._load (internal/modules/cjs/loader.js:840:27)
2020-04-29T18:05:54.318246+00:00 app[web.1]: at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
2020-04-29T18:05:54.318246+00:00 app[web.1]: at internal/main/run_main_module.js:18:47 {
2020-04-29T18:05:54.318247+00:00 app[web.1]: code: 'MODULE_NOT_FOUND',
2020-04-29T18:05:54.318247+00:00 app[web.1]: requireStack: []
2020-04-29T18:05:54.318247+00:00 app[web.1]: }
2020-04-29T18:05:54.322818+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2020-04-29T18:05:54.323019+00:00 app[web.1]: npm ERR! errno 1
2020-04-29T18:05:54.323952+00:00 app[web.1]: npm ERR! @ start: `node dist/src/index.js`
2020-04-29T18:05:54.324051+00:00 app[web.1]: npm ERR! Exit status 1
2020-04-29T18:05:54.324180+00:00 app[web.1]: npm ERR!
2020-04-29T18:05:54.324265+00:00 app[web.1]: npm ERR! Failed at the @ start script.
2020-04-29T18:05:54.324352+00:00 app[web.1]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
 		"compile:check": "tsc --project . --noEmit",
 		"test": "jest",
 		"test:watch": "jest --watch",
-		"start": "node dist/src/index.js",
+		"start": "node dist/index.js",
 		"ci:lint": "eslint --ext .ts --ignore-path .gitignore .",
 		"ci:format": "prettier --ignore-path .gitignore -l .",
 		"lint": "yarn run ci:lint --fix",

